host/ProtocolGateway.Host.Fabric.FrontEnd/PackageRoot/Config/FrontEnd.Mqtt.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "RetainPropertyName": "mqtt-retain",
+  "DupPropertyName": "mqtt-dup",
+  "QoSPropertyName": "mqtt-qos",
+  "MqttInboundTemplates": [ "devices/{deviceId}/messages/events" ],
+  "MqttOutboundTemplates": [ "devices/{deviceId}/messages/devicebound" ],
+  "MqttQoSStateProvider": "AzureStorage",
+  "ConnectArrivalTimeout": "00:00:00",
```

Comment:
>00:00:00 [](start = 28, length = 8)

you might want to limit time allowed for CONNECT packet to arrive by default as a good practice. Otherwise cleanup for objects backing ghost connection might never happen.